### ISSUE TITLE
States Windows 10 OSes are supported,

### ISSUE THREAD

**Issue by 1RedOne** [MicrosoftDocs/windowsserverdocs#5775]
Should these docs be updated?  I do not know if Windows 10 OSes are supported for WAC.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e80c473e-b916-0047-64f6-3c4c0eed7df6
* Version Independent ID: b70c774f-8962-0e48-8294-645c03530f40
* Content: [Windows Admin Center frequently asked questions](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/understand/faq#can-i-deploy-windows-admin-center-on-a-windows-10-computer-)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/understand/faq.yml](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/manage/windows-admin-center/understand/faq.yml)
* GitHub Login: @nwashburn-ms
* Microsoft Alias: **niwashbu**

**Comment by trungtran-msft** [comment]
Yes @1RedOne, Windows Admin Center still works with Windows 10 OS.

**Comment by 1RedOne** [comment]
Can it monitor a Windows 10 OS?

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
